pages/settings/index.js

```diff
@@ -50,10 +151,84 @@ export default function SettingsPage({
   );
 }
 
+const StyledCategoryName = styled.p`
+  font-size: 1.2rem;
+  margin: 0;
+  padding: 0 0 8px 0;
+`;
+
+const StyledDeletionWarning = styled.p`
+  font-size: 0.8rem;
+  margin: 0;
+  padding: 18px 0 8px 0;
+`;
+
+const StyledConfirmActionContainer = styled.div`
```

Comment:
Maybe you could use sections or other semantics instead of div ?

Reply:
Your right, we use a lot of div's in our code, as we are often unsure what would be semantically correct and since we can't decide on a tag that is fitting we usually fall back onto a classic div. We will work on that!